tokio/src/io/driver/mod.rs

```diff
@@ -197,6 +201,8 @@ impl Driver {
             return;
         }
 
+        self.inner.metrics.incr_ready_count();
```

Comment:
Do we want to do this at the top of this method to capture invalid tokens as well? or do we want this to represent only valid wakeups?